easybuild/easyblocks/s/siesta.py

```diff
@@ -0,0 +1,320 @@
+            apply_regex_substitutions(makefile, regex_subs_UtilLDFLAGS)
+
+            run_cmd('./build_all.sh', log_all=True, simple=True, log_output=True)
+
+            # Now move all the built utils to the temp installdir
+            expected_utils = [
+                'Bands/eigfat2plot',
+                'CMLComp/ccViz',
+                'Contrib/APostnikov/eig2bxsf', 'Contrib/APostnikov/rho2xsf',
+                'Contrib/APostnikov/vib2xsf', 'Contrib/APostnikov/fmpdos',
+                'Contrib/APostnikov/xv2xsf', 'Contrib/APostnikov/md2axsf',
+                'COOP/mprop', 'COOP/fat',
+                'Denchar/Src/denchar',
+                'DensityMatrix/dm2cdf', 'DensityMatrix/cdf2dm',
+                'Eig2DOS/Eig2DOS',
+                'Gen-basis/ioncat', 'Gen-basis/gen-basis',
+                'Grid/cdf2grid', 'Grid/cdf_laplacian', 'Grid/cdf2xsf',
+                'Grid/grid2cube',
+                'Grid/grid_rotate', 'Grid/g2c_ng', 'Grid/grid2cdf', 'Grid/grid2val',
+                'Helpers/get_chem_labels',
+                'HSX/hs2hsx', 'HSX/hsx2hs',
+                'JobList/Src/getResults', 'JobList/Src/countJobs',
+                'JobList/Src/runJobs', 'JobList/Src/horizontal',
+                'Macroave/Src/macroave',
+                'ON/lwf2cdf',
+                'Optimizer/simplex', 'Optimizer/swarm',
+                'pdosxml/pdosxml',
+                'Projections/orbmol_proj',
+                'SiestaSubroutine/FmixMD/Src/driver',
+                'SiestaSubroutine/FmixMD/Src/para',
+                'SiestaSubroutine/FmixMD/Src/simple',
+                'STM/simple-stm/plstm', 'STM/ol-stm/Src/stm',
+                'VCA/mixps', 'VCA/fractional',
+                'Vibra/Src/vibra', 'Vibra/Src/fcbuild',
+                'WFS/info_wfsx', 'WFS/wfsx2wfs',
+                'WFS/readwfx', 'WFS/wfsnc2wfsx', 'WFS/readwf', 'WFS/wfs2wfsx',
+            ]
+
+            if LooseVersion(self.version) <= LooseVersion("4.0"):
+                expected_utils.extend([
+                    'Bands/new.gnubands',
+                    'TBTrans/tbtrans',
+                ])
+
+            if LooseVersion(self.version) >= LooseVersion("4.0"):
+                expected_utils.extend([
+                    'SiestaSubroutine/ProtoNEB/Src/protoNEB',
+                    'SiestaSubroutine/SimpleTest/Src/simple_pipes_parallel',
+                    'SiestaSubroutine/SimpleTest/Src/simple_pipes_serial',
+                    'Sockets/f2fmaster', 'Sockets/f2fslave',
+                ])
+
+            if LooseVersion(self.version) >= LooseVersion("4.1"):
+                expected_utils.extend([
+                    'Bands/gnubands',
+                    'Grimme/fdf2grimme',
+                    'SpPivot/pvtsp',
+                    'TS/ts2ts/ts2ts', 'TS/tshs2tshs/tshs2tshs', 'TS/TBtrans/tbtrans',
+                ])
+
+            for f in expected_utils:
```

Comment:
@akesandgren don't use single-letter variable names outside of list comprehensions please, `for util in expected_utils:`